File: LinkedList/LinkedList Implementation/Node.java
//Implementation of singly linked list

class Node {
    int data;
    Node next;

    public Node(int data) {
        this.data = data;
        next = null;
    }

    public void append(int data) {

        Node n = this;
        while(n.next != null) {
            n = n.next;
        }
        n.next = new Node(data);
    }

    public void removeAt(int index) {
        int i = 0;
        Node prev = this;
        while(i + 1 < index) {
            prev = prev.next;
        }

        prev.next = prev.next.next.next;
    }
}

File: LinkedList/Exercises/Node.py
class Node:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next

    def visualizer(self):
        node = self
        s = ""
        while(node):
            if node.next == None:
                s += str(node.val)
            else:
                s += str(node.val) + " -> "

            node = node.next
        
        return s